webpack/strategies/docs.js

```diff
@@ -27,7 +27,6 @@ export default (config, options) => {
               jsLoader = value.loader;
 
               return _.extend({}, value, {
-                loader: jsLoader + '!client',
```

Comment:
As I understand we can remove `"client-loader": "0.0.1",` from `package.json` ?
I didn't downloaded this PR yet.
Only looking through diffs on GitHub.
(and you've already made three fixes :smile: what caused me to refresh this page three times)